src/Pages/LeaderBoard.js

```diff
@@ -32,21 +31,14 @@ async function navigateToProtolPage(i, name){
 
 function LeaderBoard() {
 
-  var filterStr
+  const [searchTerm, setSearchTerm] = useState("")
 
   navigate = useNavigate()
-  const [data, getData] = useState([])
 
   useEffect(() => {
-      fetchData()
       sortData()
   }, []);
```

Comment:
may want to pass in searchTerm in the array here so that it resorts the data each time the user types in the search form